src/gluonnlp/model/bert.py

```diff
@@ -342,12 +417,11 @@ def __init__(self, encoder, vocab_size=None, token_type_vocab_size=None, units=N
                  embed_size=None, embed_dropout=0.0, embed_initializer=None,
                  word_embed=None, token_type_embed=None, use_pooler=True, use_decoder=True,
                  use_classifier=True, use_token_type_embed=True, prefix=None, params=None):
-        super(BERTModel, self).__init__(prefix=prefix, params=params)
+        super().__init__(prefix=prefix, params=params)
```

Comment:
Stupid question: is not specifying parent class not required and recommended?

Reply:
Only needed in Python 2

Reply:
> Also note that, aside from the zero argument form, super() is not limited to use inside methods. The two argument form specifies the arguments exactly and makes the appropriate references. *The zero argument form only works inside a class definition, as the compiler fills in the necessary details to correctly retrieve the class being defined, as well as accessing the current instance for ordinary methods.*
https://docs.python.org/3/library/functions.html#super